src/httprpc.cpp

```diff
@@ -240,7 +240,8 @@ bool StartHTTPRPC()
     // ifdef can be removed once we switch to better endpoint support and API versioning
     RegisterHTTPHandler("/wallet/", false, HTTPReq_JSONRPC);
 #endif
-    assert(EventBase());
+    auto eventBase = EventBase();
+    assert(eventBase);
     httpRPCTimerInterface = MakeUnique<HTTPRPCTimerInterface>(EventBase());
```

Comment:
nit: Could reuse `eventBase` on this line. The type `struct event_base*` could be informative, given it makes the assert's meaning clear.

Reply:
I agree. If we assign this to a variable anyway, we should use that.